### PR TITLE
inverse_stft_window_fn

### LINKED ISSUE #21889: inverse_stft_window_fn

### PR DESCRIPTION
**Pull request by teckno**
closes #21889

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy/functional/frontends/tensorflow/signal.py [hunk 1]
```diff
@@ -34,3 +36,10 @@ def vorbis_window(window_length, dtype=ivy.float32, name=None):
 def idct(input, type=2, n=None, axis=-1, norm=None, name=None):
     inverse_type = {1: 1, 2: 3, 3: 2, 4: 4}[type]
     return ivy.dct(input, type=inverse_type, n=n, axis=axis, norm=norm)
+
+@to_ivy_arrays_and_back
+def inverse_stft_window_fn(frame_step, forward_window_fn=tf.signal.hann_window,name=None):
+    return lambda frame_length: tf.signal.inverse_stft_window_fn(frame_length, forward_window_fn, frame_step)
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/tensorflow/signal.py:42]: We don't use any framework in the frontends only Ivy functions are used to avoid dependency

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @teckno 

Check the comments below
Thanks

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by Sarvesh-Kesharwani** [comment]
Hi @teckno, kindly focus on incorporating a singular function per pull request. Specifically, in this pull request, concentrate solely on the implementation of the inverse_stft_window_fn function. To enhance clarity and maintain consistency, eliminate any alterations made in files other than signal.py and test_signal.py.
Thanks for the contribution😊.

**Pull request closed without merging**